lib/reporters/json-reporter.js

```diff
@@ -34,7 +34,7 @@ var branchCoverage = function(fileData) {
     for (var j = 0; j < branches.length; j++) {
       if (_.isArray(branches[j])) {
         for (var k = 0; k < branches[j].length; k++) {
-          if (_.isArray(branches[j][k])) {
+          if (_.isObject(branches[j][k])) {
```

Comment:
Good question, and in this reporter you're right--it's being treated as an array. But because blanket stores additional properties (consequent and alternate) on the array it generates at this level, I had to switch this to be an object that looks like an array (its properties are 0, 1, consequent, and alternate) so that those additional properties would be serialized across the wire when posting to the reporter. That way, reporters would have access to them if needed--the HTML reporter in particular uses those additional properties, and others may in the future.